### ISSUE TITLE
Many buttons fail accessibility check

### ISSUE THREAD

**Issue by Omikorin** [naugtur/meetjs.pl#234]
And they fail really awful...

Color contrast check [here](https://webaim.org/resources/contrastchecker/?fcolor=FAFAFA&bcolor=BCD35D)

<img width="748" height="814" alt="Image" src="https://github.com/user-attachments/assets/61169561-ea5e-40a2-92fa-497015f4f8be" />

**Comment by dzienisz** [comment]
I don't think it's such a problem for us. Can you propose any quick-fix solution?

**Comment by Omikorin** [comment]
I think it's important for our visitors. I have trouble reading those buttons. Likewise, I checked for `#0A0A0A` and other black colors, and it's an excellent choice. I sent a PR with `black` from defaults because I don't know how this tailwoind works, and I don't feel updating a random place to overwrite foreground texts by mistake.
